### ISSUE TITLE
restore the old podcast page

### ISSUE THREAD

**Issue by Sean1807** [vgmdb/tracker#324]
Carl recently uploaded all the episodes to his youtube channel. https://www.youtube.com/user/SpiritOfGamer/videos

**Comment by Illidanz** [comment]
The youtube link can replace the podcast URL in the threads in this subforum: https://vgmdb.net/forums/forumdisplay.php?f=13
It can be accessed by just editing the first post in the thread.

The old page is still active here and would only require some slight editing: https://vgmdb.net/cast

I'm not sure if I would restore it where it was before though considering the podcast is inactive. Any suggestion is welcome.

**Comment by vgmcal** [comment]
It wasn't just the dead links that were a factor in removing it. The podcast has been inactive for a decade now.

I wouldn't support putting it back in the nav bar, but linking it through the forums is certainly acceptable.

**Comment by nstz0** [comment]
I agree with cal, displaying a podcast 10 years out of date in the header doesn't seem relevant.

**Comment by Sean1807** [comment]
i think the important part is to make it easily accessible to people who might find it interesting. whether it's active or inactive doesn't seem relevant.

**Comment by vgmcal** [comment]
The forum is easily accessible, so that should work then.
